### PR TITLE
ESM-70 - Fix fatal when moving tickets in ET

### PR DESCRIPTION
**Pull request by jesseeproductions**
-When moving tickets in ET/ET+ there is a fatal error caused in the two changed methods
-The methods expect a string, but in the move modal $page is null

Artifacts
-none

_Ref:_ [ESM-70](https://stellarwp.atlassian.net/browse/ESM-70)

[ESM-70]: https://stellarwp.atlassian.net/browse/ESM-70?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Uplink/Admin/Plugins_Page.php [hunk 1]
```diff
@@ -32,7 +32,7 @@ class Plugins_Page {
 	 *
 	 * @return void
 	 */
-	public function display_plugin_messages( string $page ): void {
```
Comment (jesseeproductions) [inline src/Uplink/Admin/Plugins_Page.php:35]: @borkweb Matt, these type check are causing fatals when trying to move tickets as $page is null. 

I removed them, but I could also do a check in the provider instead and leave the the type check.

Comment (defunctl) [reply src/Uplink/Admin/Plugins_Page.php:35]: @jesseeproductions sorry, this is my fault for trusting the existing docblocks when they were clearly wrong. I think type juggling in the provider is a great idea, however if you want to leave it this way, please update the `@param` dockblock to show the correct type coming in: `string|null`

Comment (defunctl) [reply src/Uplink/Admin/Plugins_Page.php:35]: @jesseeproductions mind if I take a stab at just moving the type juggling into the provider?

Comment (jesseeproductions) [reply src/Uplink/Admin/Plugins_Page.php:35]: No Worries @defunctl I updated the docblocks. 

My preference is to leave it this way and not have logic in the Provider

Comment (defunctl) [reply src/Uplink/Admin/Plugins_Page.php:35]: All good, I did add this which also fixes the tests/PHPStan issues (separate from this): https://github.com/stellarwp/uplink/pull/56

**Comment by kadencewp** [comment]
Duplicate: #53

**Pull request merged**